newIDE/app/src/ProjectCreation/CreateProject.js

```diff
@@ -82,6 +82,20 @@ export const createNewProjectFromTutorialTemplate = (
   return newProjectSource;
 };
 
+export const createNewProjectFromCourseChapterTemplate = (
+  templateUrl: string,
+  courseChapterId: string
+): NewProjectSource => {
+  // TODO: Send event?
```

Comment:
Let's add another property in new game created which can be "isCourseChapterTemplate: true".

This will make analysis easier/way more "bullet proof" than doing fragile regexes.